src/components/Modals/LoginModal/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react'
+import BaseModal from '../index'
+import Logo from '@/assets/img/logo.png'
+import * as S from '../styles'
+import RegisterModal from '../RegisterModal'
```

Comment:
Existe um padrão pra organização de imports. Geralmente é normal a gente organizar os imports nessa ordem: `built-in modules`, `installed modules`, `created modules`. 

Geralmente, quando se trabalha com styled componentes, se coloca o `import * as S` como último import.

Exemplo:
```typescript
import React, { useState } from 'react'

import RegisterModal from '../RegisterModal'
import BaseModal from '../index'
import Logo from '@/assets/img/logo.png'

import * as S from '../styles'
```

Outra boa prática é evitar o uso de `imports relativos`. O javascript e seus derivados tem um problema muito grande chamado de `relative path hell`... Há algumas formas de contornar isso. Nesse teu caso ainda não é um relative path hell, mas acho que é uma boa manter o import relativo somente para arquivos que estão no mesmo nível ou em níveis maiores do que o arquivo em questão. Por exemplo:

```typescript
import React, { useState } from 'react'

import RegisterModal from '@/components/Modals/RegisterModal'
import BaseModal from '@/components/Modals'
import Logo from '@/assets/img/logo.png'

import * as S from '@/components/Modals/styles'
```

Reply:
Que massa! Obrigado por compartilhar! Feito em https://github.com/Pews-DEV/book-writer-frontend/pull/12/commits/a2d37773311c13a9158605c361bc5b0b0ec5372b